extensions/amp-access/validator-amp-access.protoascii

```diff
@@ -32,7 +32,6 @@ tags: {  # amp-access (json)
   unique: true
   mandatory_parent: "HEAD"
   requires_extension: "amp-access"
-  requires_extension: "amp-analytics"
```

Comment:
Can you change validator-amp-analytics.protoascii from

`requires_usage: NONE` to `requires_usage: GRANDFATHERED`

This will cause the inclusion of amp-analytics extension to emit a *warning* if the `<amp-analytics>` tag is not present. Will not cause any errors, so will not cause any pages to newly fail validation.

Reply:
Done. PTAL.